src/osip-monitoring-view/pom.xml

```diff
@@ -29,5 +29,11 @@
 			<artifactId>jfxtras-gauge-linear</artifactId>
 			<version>8.0-r6-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>org.testfx</groupId>
+			<artifactId>testfx-junit</artifactId>
+			<version>4.0.1-alpha</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
Würde die Dependency in die haupt pom machen, so wie junit auch. Wobei David die gleichen Commits drin hat, das gibt wahrscheinlich Konflikte